platform/platform-configuration-listener/src/test/java/org/codice/ddf/configuration/store/SystemConfigurationMigratorTest.java

```diff
@@ -0,0 +1,336 @@
+import static org.powermock.api.mockito.PowerMockito.doThrow;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.verifyStatic;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.rule.PowerMockRule;
+
+@PrepareForTest({SystemConfigurationMigrator.class, FileUtils.class})
+public class SystemConfigurationMigratorTest {
+
+    @Rule
+    public PowerMockRule rule = new PowerMockRule();
+
+    private static final Path DDF_HOME = Paths.get("/my/ddf/home");
+
+    private static final Path EXPORT_DIRECTORY = Paths.get("/my/ddf/home/etc/exported");
+
+    private static final String KEYSTORE_REL_PATH = "etc/keystores/keystore.jks";
+
+    private static final String KEYSTORE_INVALID_PATH = "/my/ddf/home/../../etc/keystores/keystore.jks";
+
+    private static final String KEYSTORE_ABS_PATH = "/my/ddf/home/etc/keystores/keystore.jks";
+
+    private static final String TRUSTSTORE_REL_PATH = "etc/keystores/truststore.jks";
+
+    private static final String TRUSTSTORE_INVALID_PATH = "/my/ddf/home/../../etc/keystores/truststore.jks";
+
+    private static final String TRUSTSTORE_ABS_PATH = "/my/ddf/home/etc/keystores/truststore.jks";
+
+    private static final String ENC_PROP_FILE_REL_CRL_PATH = "src/test/resources/encryptionCrlRelPath.properties";
+
+    private static final String ENC_PROP_FILE_ABS_CRL_PATH = "src/test/resources/encryptionCrlAbsPath.properties";
+
+    private static final String ENC_PROP_FILE_INVALID_CRL_PATH = "src/test/resources/encryptionCrlInvalidPath.properties";
+
+    private static final String KEYSTORE_SYSTEM_PROP = "javax.net.ssl.keyStore";
+
+    private static final String TRUSTSTORE_SYSTEM_PROP = "javax.net.ssl.trustStore";
+
+    private static final String WS_SECURITY_DIR_REL_PATH = "etc/ws-security";
+
+    private static final String PDP_POLICIES_DIR_REL_PATH = "etc/pdp";
+
+    private static final String SYSTEM_PROPERTIES_REL_PATH = "etc/system.properties";
+
+    private static final String USERS_PROPERTIES_REL_PATH = "etc/users.properties";
+
+    private static final String VALID_CRL_REL_PATH_1 = "etc/certs/demoCA/crl/crl.pem";
+
+    @Before
+    public void setup() {
+        mockStatic(System.class);
+        mockStatic(FileUtils.class);
+    }
+
+    @Test
+    public void testExportValidRelativePaths() throws Exception {
+        // Setup
+        when(System.getProperty(KEYSTORE_SYSTEM_PROP)).thenReturn(KEYSTORE_REL_PATH);
```

Comment:
Since we're only mocking `System` for the `getProperty()` method, couldn't we just call `System.setProperty()` here and _not_ mock `System`?